include/z64cutscene.h

```diff
@@ -344,6 +344,42 @@ typedef enum CutsceneDestination {
     /* 0x77 */ CS_DEST_ZELDAS_COURTYARD_RECEIVE_LETTER
 } CutsceneDestination;
 
+/*
+correlation with `enum SceneLayer`:
+  CS_INDEX_MANUAL:     [SCENE_LAYER_CHILD_DAY .. SCENE_LAYER_ADULT_NIGHT]
+  CS_INDEX_SCRIPTED_*: SCENE_LAYER_CUTSCENE_FIRST
+
+correlation with `enum EntranceIndex`
+  CS_INDEX_MANUAL:     base entrance index + scene layer
+  CS_INDEX_SCRIPTED_*: base entrance index + SCENE_LAYER_CUTSCENE_FIRST + (cutscene index & 0xF)
+
+`z_select.c` naming:
+  CS_INDEX_NONE:       "Stage: night"
+  CS_INDEX_LAST:       "Stage: day"
+  CS_INDEX_SCRIPTED_*: "Stage demo 0*", between [0xFFF0 .. 0xFFFA]
+*/
+typedef enum CutsceneIndex {
+    CS_INDEX_NONE = 0x0000,
+    CS_INDEX_MANUAL = 0xFFEF,
+    CS_INDEX_SCRIPTED_0 = 0xFFF0,
```

Comment:
I don't particularly like spelling out "SCRIPTED" here because I believe there's no mechanism to play unscripted cutscenes using this value.

Reply:
agreed, i'm compiling a commit with suggested name changes